src/arkouda_server.chpl

```diff
@@ -159,7 +159,7 @@ proc main() {
         if (memTrack) {writeln("bytes of memoryUsed() = ",memoryUsed()); try! stdout.flush();}
 
         // end timer for command processing
-        if v{writeln("<<< ", cmd," took ", Time.getCurrentTime() - t1,"sec"); try! stdout.flush();}
+        if v{writeln("<<< %s took %.17r sec".format(cmd, Time.getCurrentTime() - t1)); try! stdout.flush();}
```

Comment:
The `Time.getCurrentTime()` function returns the number of elapsed (default) seconds since midnight local time, not Unix epoch.

In my opinion, this function is not generally useful; actually it is deceiving and more of an anti-pattern such as in this usage here. Subtracting between two `Time.getCurrentTime()` timepoints will roll over if you time something across midnight and you'll get negative elapsed time. You should use the `Timer` class provided by the `Time` module instead of this function.

Reply:
ill make the change for the main timer

Reply:
@BryantLam change is made and pushed.